### ISSUE TITLE
Mockup: Configuration > Settings > General

### ISSUE THREAD

**Issue by ggeisler** [samvera/hyku#438]
Change current menu item Configuration > Settings > Labels to Configuration > Settings > General, where the General page has tabs for distinct sets of general repository settings:

## Labels tab

(TBD: strategy for indicating required fields that is more restrained than is currently in Sufia.)

![labels-tab](https://cloud.githubusercontent.com/assets/101482/19823998/f02ab7b4-9d20-11e6-9371-53fc72d29704.png)

## Analytics tab
- Question: Is this tab needed? Will the repository be providing their own GA id? 
_Based on discussion in #460, the answer seems to be Yes_

So, at least (only) an input for entering the GA key/ID

## Other tabs
TBD. No current need for Notifications or Repo Objects tabs. Mockup to be revised.

**Comment by mjgiarlo** [comment]
@jcoyne @hannahfrost @ggeisler Do you think this design is good to go? I believe analytics support is required for MVP.

**Comment by ggeisler** [comment]
@mjgiarlo Technically this was still in the Design Needed column (until you moved it to Ready a few hours ago) because the design was not complete ("Mockup to be revised"). It had not been prioritized near the top of the Design Needed queue so I didn't know this should take priority over other issues. I'm concerned that @hannahfrost and I developed what seems to be a decent structured requirements and design process, but that this process is either not understood or respected by the rest of the team (cf. the whole debate last week about the implementation of the select work type for batch upload).

That said, if you're looking for tickets for developers to work on, this is probably fine without further design. Based on comments I made above, the Analytics tab should have a labeled input field for a GA ID, and unlike the mockup, there appears to be no need for the Notifications or Repository Objects tabs. Seems straightforward enough. (I would have just preferred that this had been elevated to the top of the design needed queue earlier so I could have provided a finished mockup, trivial as the updates might be.)

**Comment by mjgiarlo** [comment]
@ggeisler I was tweaking labels here in GitHub -- I didn't realize that adding `HyBox ready` would override `design needed`. Should have double-checked in Waffle after I made that change. We have a fine process, and I doubt anyone is intentionally undermining it; it's more that we're working on a complex project with a lot of moving parts across a number of different codebases. I've undone that change. Sorry for that.

**Comment by ggeisler** [comment]
No problem, @mjgiarlo. I don't mean to make a big deal about it. I do understand this is a complex project and we're never going to have as smooth a process as we have had on smaller projects. I just want to prioritize my HyBox time to provide the best benefit to the project and it seems hard to get a handle on what that is sometimes.

**Comment by mjgiarlo** [comment]
@ggeisler Absolutely, @ggeisler. I suspect we need to spend more time on team coordination in the new year; we will all benefit from that.

**Comment by hannahfrost** [comment]
At this stage, we are deprioritizing analytics from the MVP scope. (Lots of good work done towards spec'ing it and the tasks to make it happen, so we can pick it up soon.) Given that, let's implement the design shown in this mockup but just with the one tab: General. We'll add Analytics when we are ready to implement that functionality.  

Sound good, @ggeisler @mjgiarlo ?

**Comment by mjgiarlo** [comment]
Sounds good, @hannahfrost!

**Comment by ggeisler** [comment]
Sounds fine to me. But note that we can probably just leave the tab label as it currently is, "Labels," rather than "General."

**Comment by hannahfrost** [comment]
Revisit this ticket when functionality for the other tabs is ready to be made available.
